src/index.html

```diff
@@ -3,11 +3,640 @@
+            <div class="section4__box section4__box-1">
+              <img
+              class="section4__box-image"
+              alt="what-say-man"
+              src="./images/section/what-say-man.png"
+              >
+
+              <img
+              class="section4__box--dots-image"
+              alt="dots"
+              src="./images/section/dots.png"
+              >
+
+              <p class="section4__box-subtext">
+                AIR’s ideas are refreshing and out of the box.
+                Authentic team that focuses on the important path of the brand.
+              </p>
+
+              <p class="section4__subtitle-text">Tal Gilad</p>
+              <p class="section4__subtitle-bottom">Teach for America</p>
+            </div>
+
+            <div class="section4__box section4__box-2">
+              <img
+              class="section4__box-image"
+              alt="what-say-man"
+              src="./images/section/what-say-woman.png"
+              >
+
+              <img
+              class="section4__box--dots-image"
+              alt="dots"
+              src="./images/section/dots.png"
+              >
+
+              <p class="section4__box-subtext">
+                AIR is an exceptional agency that leads with creative talent,
+                first-class account servicing.
+              </p>
+
+              <p class="section4__subtitle-text">Azadeh Hawkins</p>
+              <p class="section4__subtitle-bottom">Hawkins Consulting</p>
+            </div>
+
+            <div class="section4__box section4__box-3">
+              <img
+              class="section4__box-image"
+              alt="what-say-man"
+              src="./images/section/what-say-teenager.png"
+              >
+
+              <img
+              class="section4__box--dots-image"
+              alt="dots"
+              src="./images/section/dots.png"
+              >
+
+              <p class="section4__box-subtext">
+                AIR raises the agency bar to stratospheric
+                heights on both creative output and client service.
+              </p>
+
+              <p class="section4__subtitle-text">Michel Grover</p>
+              <p class="section4__subtitle-bottom">Hulu</p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="section5">
+        <div class="section5__container">
+          <h1 class="section5__title">Vision, Passion, Results</h1>
+          <h2 class="section5__text">
+            We are sure that first-rate job is possible
+            only if all three components are united.
+          </h2>
+        </div>
+
+        <a class="section5__button" href="#contact">
+          <button class="section5__button-content">Apply</button>
+        </a>
+      </section>
+    </main>
+
+    <footer class="footer" id="contact">
+      <div class="footer__content">
+        <div class="footer__container">
+          <h1 class="footer__title">Send us a message</h1>
```

Comment:
This tag is used correctly only 1 time per page and in one place, so it needs to be fixed everywhere
```suggestion
          <h3 class="footer__title">Send us a message</h3>
```